examples/snes_controller_simpletest

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: 2021 John Furcean
+# SPDX-License-Identifier: MIT
+
+# Classic Controller also work with CLV-202.
+# But the "Super Nintendo SNES Classic Mini Controller" has less button and not stick.
+
+import board
+from wiichuck.classic_controller import ClassicController
+
+controller = ClassicController(board.I2C())
+
+while True:
+    joysticks, buttons, dpad, triggers = controller.values
```

Comment:
Should probably be one of the following instead.

```python
_, buttons, dpad, _ =    controller.values
```
or
```python
buttons = controller.buttons
dpad = controller.dpad
```

Reply:
Took the first option and it works.